### PR TITLE
BI-10192 adding post method to active officers details controller

### PR DESCRIPTION
**Pull request by SVarallo**

### CODE REVIEW

File: src/controllers/tasks/active.officers.controller.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
       const residentialAddress = formatAddressForDisplay(activeOfficerDetails?.residentialAddress);
       return res.render(Templates.ACTIVE_OFFICERS, {
         backLinkUrl: urlUtils.getUrlToPath(TASK_LIST_PATH, req),
-        officerErrorMsg: DIRECTOR_DETAILS_ERROR,
+        officerErrorMsg: OFFICER_DETAILS_ERROR,
```
Comment (markpit) [inline src/controllers/tasks/active.officers.controller.ts:69]: this controller is for the current live version with 1 director only. Do we know if that error message is supposed to change from director to officer for the current live version?

Comment (SVarallo) [reply src/controllers/tasks/active.officers.controller.ts:69]: Admittedly no, shall we just add a new error message for officers then and just leave the directors error as is?

Comment (markpit) [reply src/controllers/tasks/active.officers.controller.ts:69]: I would create new message, unless we have had a requirement to change the existing one then keep them separate

**Review by markpit — APPROVED** [review]

File: views/tasks/active-officers-details.html [hunk 2]
```diff
@@ -65,8 +71,8 @@ <h1 class="govuk-heading-l">Check the officers' details</h1>
       {{ govukRadios({
         classes: "govuk-radios",
         idPrefix: "officers",
-        name: "officers",
-        errorMessage: directorDetailsError,
+        name: "activeOfficers",
```
Comment (mwestacott) [inline views/tasks/active-officers-details.html:74]: This is fine but also delivers what I was working on (not very much!) - https://companieshouse.atlassian.net/browse/BI-10191. So we can mark that task as done too.

Comment (SVarallo) [reply views/tasks/active-officers-details.html:74]: Should still need the data event ids added, I can put that in this as well if that's easier?

**Pull request merged**